apps/site/assets/ts/schedule/components/ScheduleDirection.tsx

```diff
@@ -147,8 +155,9 @@ const ScheduleDirection = ({
         );
       }
     },
+    // only re-run the effect if any of these things change
```

Comment:
@hannahpurcell maybe these comments are useful because this code is so distant from where the `useEffect` hook is first defined, but I think this comment is just describing the default behavior of `useEffect`